features/networking/sgw-lgw-migration-externalIP.feature

```diff
@@ -0,0 +1,84 @@
+Feature: SGW<->LGW migration related scenarios
```

Comment:
Not sure why we are creating this as a different feature file. May be Just add this as a third scenario in same ```sgw-lgw-migration.feature```? This way it will always run as a separate use case

Reply:
> Not sure why we are creating this as a different feature file. May be Just add this as a third scenario in same `sgw-lgw-migration.feature`? This way it will always run as a separate use case

@anuragthehatter That's sounds a good workaround,I will give it a try, thanks!